### PR TITLE
[DOC-6594]: Help Wanted: Update Mobile Troubleshooting Docs

### LINKED ISSUE #6594: Help Wanted: Update Mobile Troubleshooting Docs
## Context

The Mobile apps FAQ documentation doesn't specify where users can access notification message logs for their Mattermost mobile app.

## Docs request

Update the [Mobile apps FAQ product documentation](https://docs.mattermost.com/deploy/mobile-faq.html) to add the following FAQ:

```
Question: Where can I find mobile message notification logs?
Answer: Notification messages are logged to the notifications.log file.
```

## How you can help

Confirm the path to access the ``notification.log`` file, and create a docs PR to update the documentation with these details that includes the path to the log file.

### PR DESCRIPTION
**Pull request by Rutam21**
## Summary

This commit adds a new section in the Mobile FAQ page that answers the following question.

```bash
Where can I find mobile message notification logs?
```

## Ticket Link

This PR fixes #6594 .

### CODE REVIEW

**Comment by Rutam21** [comment]
@cwarnermm Please review and suggest changes, if any. Thanks.

File: source/deploy/mobile-faq.rst [hunk 1]
```diff
@@ -372,3 +372,15 @@ In these scenarios, the ``login`` audit log won't have the ``deviceId``, and the
 If the app receives the ``deviceId`` later, the device will send the new ``deviceId`` to the server, generating an ``attachDeviceId`` audit log, and adding the ``deviceId`` to the session data in the database.
 
 Since the ``deviceId`` relates to the application, connections through the web browser, even on mobile, won't have a ``deviceID``.
+
+Where can I find mobile message notification logs?
+-------------------------------------------------------------
+Notification messages are logged to the ``notifications.log`` file. However, this file can only be accessed by server admins.
+Generally you will find it by navigating to ``Notification Settings--->File Location``. One thing to note here is that the ``notifications.log`` exists only if you have enabled it in the Notification Settings i.e., ``NotificationSettings.EnableFile`` should be set to ``TRUE``.
```
Comment (cwarnermm) [inline source/deploy/mobile-faq.rst:379]: Where would an admin access ``Notification Settings > File Location`` specifically?

Comment (cwarnermm) [inline source/deploy/mobile-faq.rst:379]:
```suggestion
System admins can access the ``notifications.log`` file by going to ``Notification Settings > File Location``. The ``notifications.log`` file exists only when enabled in Mattermost by setting ``NotificationSettings.EnableFile`` to ``true`` in the ``config.json`` file.
```

**Review by cwarnermm — CHANGES_REQUESTED** [review]
Thanks, @Rutam21! I've provided feedback inline.

File: source/deploy/mobile-faq.rst (hunk 1 above)
Comment (Rutam21) [reply source/deploy/mobile-faq.rst]: Thanks for pointing this out. Will modify this to add better details.

**Comment by Rutam21** [comment]
@cwarnermm Necessary changes have been done. Please review it again. Thanks.

File: source/deploy/mobile-faq.rst [hunk 2]
```diff
@@ -372,3 +372,16 @@ In these scenarios, the ``login`` audit log won't have the ``deviceId``, and the
 If the app receives the ``deviceId`` later, the device will send the new ``deviceId`` to the server, generating an ``attachDeviceId`` audit log, and adding the ``deviceId`` to the session data in the database.
 
 Since the ``deviceId`` relates to the application, connections through the web browser, even on mobile, won't have a ``deviceID``.
+
+Where can I find mobile message notification logs?
+-------------------------------------------------------------
+Notification messages are logged to the ``notifications.log`` file.
+System admins can only access this ``notifications.log`` file by going to the ``File Location`` set in the ``Notification Settings``. The admins also have the privilege to set this file location and in case, no file location has been set, then the ``notifications.log`` file gets stored in the default working directory.
```
Comment (cwarnermm) [inline source/deploy/mobile-faq.rst:379]:
```suggestion
System admins must enable notification logs in the ``config.json`` file by setting ``EnableFile`` to ``true``, and specifying an optional file location via ``FileLocation``. When no location is configured, the ``notifications.log`` file is stored in the default Mattermost directory. See the `logging configuration settings <https://docs.mattermost.com/configure/environment-configuration-settings.html#logging>`__ documentation for details.
```

File: source/deploy/mobile-faq.rst [hunk 3]
```diff
@@ -372,3 +372,16 @@ In these scenarios, the ``login`` audit log won't have the ``deviceId``, and the
...
+Also, the ``notifications.log`` file exists only when it is enabled in Mattermost by setting ``NotificationSettings.EnableFile`` to ``true`` in the ``config.json`` file.
```
Comment (cwarnermm) [inline source/deploy/mobile-faq.rst:380]: This text has been incorporated into the proposed text directly above and is no longer needed. Please remove.

File: source/deploy/mobile-faq.rst [hunk 4]
```diff
@@ -372,3 +372,16 @@ In these scenarios, the ``login`` audit log won't have the ``deviceId``, and the
...
+You can learn more about logging `here <https://docs.mattermost.com/configure/environment-configuration-settings.html#logging>`__.
```
Comment (cwarnermm) [inline source/deploy/mobile-faq.rst:381]: This text has been incorporated into the proposed text directly above and is no longer needed. Please remove.

File: source/deploy/mobile-faq.rst [hunk 5]
```diff
@@ -372,3 +372,16 @@ In these scenarios, the ``login`` audit log won't have the ``deviceId``, and the
...
+
+The team members / users can access their notification logs based on their device platform.
+Android users can view the logs using ``logcat``.
+iOS users can acess the logs in the ``console app`` on their MacOS.
+However, the Android and iOS logs may differ from each other as different things are logged while receiving and processing requests based on the user platform.
```
Comment (cwarnermm) [inline source/deploy/mobile-faq.rst:386]: Let's remove this last line about the log file contents being different between platforms.

**Review by cwarnermm — CHANGES_REQUESTED** [review]
Thanks, @Rutam21! Feedback provided inline to ensure the new content aligns with existing tone/voice & formatting.

File: source/deploy/mobile-faq.rst (hunk 4 above)
Comment (Rutam21) [reply source/deploy/mobile-faq.rst]: Suggestions applied. Thanks.

File: source/deploy/mobile-faq.rst (hunk 5 above)
Comment (Rutam21) [reply source/deploy/mobile-faq.rst]: Suggestions applied. Thanks.

**Comment by Rutam21** [comment]
@cwarnermm All the suggested changes are applied. Please review. Thank you.

**Pull request merged**